src/helpers/signalHelpers.ts

```diff
@@ -19,14 +19,14 @@ export function validChannelId(channelId: string, connections: IConnectionDetail
 /**
  * Returns the configuration for the RTC peerconnection
  */
-export function getRTCConnectionConfig(type: string, turnExpiration: number, turnSecret: string, turnUrl: string): RTCConfiguration {
+export function getRTCConnectionConfig(type: string, turnExpiration: number, turnSecret: string, turnUrl: string | string[]): RTCConfiguration {
     const time = Math.floor(Date.now() / 1000);
     const username = `${time + turnExpiration}:${type}`;
     const credential = crypto.createHmac("sha1", turnSecret).update(username.toString()).digest("base64");
     return {
         iceCandidatePoolSize: 5,
         iceServers: [{
-            urls: [turnUrl],
+            urls: turnUrl,
```

Comment:
Still url's' when you can only receive one?

Reply:
![Screen Shot 2022-12-13 at 10 54 26](https://user-images.githubusercontent.com/7520694/207285906-00613f69-8e80-4285-8a3e-fba397416475.png)
I agree the naming is weird but the interface (by global webrtc) is like this :o